src/components/Navbar.tsx

```diff
@@ -8,17 +14,24 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={({ isActive }) =>
+              cn('navbar-item', isTabActive({ isActive }))
+            }
+            to="/"
+          >
             Home
-          </a>
+          </NavLink>
 
-          <a
+          <NavLink
             aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+            className={({ isActive }) =>
+              cn('navbar-item', isTabActive({ isActive }))
+            }
+            to="people"
           >
             People
-          </a>
+          </NavLink>
```

Comment:
here you forgot to add this. make sure to do this here and in all People link in table

> "Keep search params when navigating within the People page (when selecting a person or clicking the People link)."